caampr-backend/src/test/java/com/jelistan/caampr/lambda/provider/DynamoGearProviderTest.java

```diff
@@ -0,0 +1,52 @@
+package com.jelistan.caampr.lambda.provider;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBQueryExpression;
+import com.amazonaws.services.dynamodbv2.datamodeling.PaginatedQueryList;
+import com.jelistan.caampr.lambda.model.Gear;
+import com.jelistan.caampr.lambda.model.GearListRequest;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+public class DynamoGearProviderTest {
+    @Mock
+    private GearListRequest request;
+
+    @Mock
+    private DynamoDBMapper mapper;
+
+    @Mock
+    private PaginatedQueryList<Gear> queryList;
+
+    private DynamoGearProvider unit;
+
+    @Before
+    public void setUp() throws Exception {
+        MockitoAnnotations.openMocks(this);
+        queryList.add(GearProvider.FAKE_GEAR);
```

Comment:
`queryList` is a mock, so doing a `queryList.add` here doesn't serve any purpose.